charts/accountapp/values.yaml

```diff
@@ -29,8 +29,8 @@ ingress:
 #    - secretName: accounts-tls
 #      hosts:
 #        - accounts.jenkins.io
-resources: # We usually recommend not to specify default resources and to leave this as a conscious
-# choice for the user. This also increases chances charts run on environments with little
+# We usually recommend not to specify default resources and to leave this as a conscious: null # choice for the user. This also increases chances charts run on environments with little
```

Comment:
`null` or empty object? The comment seems to imply one should change the defaut `{}` value to `null`

Also, should this be splited to two lines?

Reply:
Good catch. The fix with `with` helm control flow element does not care as the resulting template pipeline is `false` wether the value is null, empty or undefined. 

PR to update the comment?